test/WebSites/RoutingWebSite/Startup.cs

```diff
@@ -26,10 +26,26 @@ public void Configure(IApplicationBuilder app)
 
             app.UseMvc(routes =>
             {
-                routes.MapRoute(
-                    "areaRoute",
-                    "{area:exists}/{controller}/{action}",
-                    new { controller = "Home", action = "Index" });
+                //routes.MapRoute(
```

Comment:
Can you update the code here? Probably don't want to check in these commented parts.